liquidation-monitor/src/agent.spec.js

```diff
@@ -0,0 +1,72 @@
+const {
```

Comment:
This looks like the starter Bot test script example.  Need to create tests for this Bot.

Reply:
Tests completed.